### PR TITLE
Fix windows_test action

### PR DESCRIPTION
**Pull request by nikteliy**
1.  wget saves the archive with an incorrect name. I set output filename to fix "The system cannot find the file specified" error. 
2.  os.kill(pid, 1) on Windows kills a process with exit code 1, which may be the cause of issue #203 . It looks like the .terminate() method kills the `mainloop` process correctly.

### CODE REVIEW

File: .github/workflows/windows.yml [hunk 1]
```diff
@@ -15,10 +15,9 @@ jobs:
         run: choco install 7zip wget python
       - name: Get snap7
         run: |
-          wget.exe --content-disposition -c https://sourceforge.net/projects/snap7/files/1.4.2/snap7-full-1.4.2.7z/download
+          wget.exe -O snap7-full-1.4.2.7z -c https://sourceforge.net/projects/snap7/files/1.4.2/snap7-full-1.4.2.7z/download
```
Comment (gijzelaerr) [inline .github/workflows/windows.yml:18]: if I look at the latest master build https://github.com/gijzelaerr/python-snap7/runs/1340797153?check_suite_focus=true

the file is saved with the right name. Are you sure you are using `wget.exe` and not the powershell alias? also the extraction works fine:

```
2020-11-02 07:41:02 (4.99 MB/s) - 'snap7-full-1.4.2.7z' saved [20591252/20591252]


7-Zip 19.00 (x64) : Copyright (c) 1999-2018 Igor Pavlov : 2019-02-21

Scanning the drive for archives:
1 file, 20591252 bytes (20 MiB)

Extracting archive: snap7-full-1.4.2.7z
```

File: .github/workflows/windows.yml [hunk 2]
```diff
@@ -15,10 +15,9 @@ jobs:
...
           7z x snap7-full-1.4.2.7z
-          Copy-Item snap7-full-1.4.2\release\Windows\Win64\snap7.dll snap7\.
-          Copy-Item snap7-full-1.4.2\release\Windows\Win64\snap7.dll C:\Windows\System32\.
+          Copy-Item snap7-full-1.4.2\release\Windows\Win64\snap7.dll C:\Windows\System32\
```
Comment (gijzelaerr) [inline .github/workflows/windows.yml:20]: we copy the dll into the folder to prepare for the binary wheel.

File: test/test_client.py [hunk 3]
```diff
@@ -37,7 +37,7 @@ def setUpClass(cls):
 
     @classmethod
     def tearDownClass(cls):
-        kill(cls.process.pid, 1)
+        cls.process.terminate()
```
Comment (gijzelaerr) [inline test/test_client.py:40]: interesting, that could be it. thanks :)

**Review by gijzelaerr — CHANGES_REQUESTED** [review]

File: .github/workflows/windows.yml (hunk 2 above)
Comment (nikteliy) [reply .github/workflows/windows.yml:20]: Are you sure it works correctly? If snap7 folder doesn't exist then a file with that name will be created. Sorry, I probably don't fully understand.
copy-item-test

File: .github/workflows/windows.yml (hunk 1 above)
Comment (nikteliy) [reply .github/workflows/windows.yml:18]: I noticed an error at this build: https://github.com/gijzelaerr/python-snap7/runs/1457872062?check_suite_focus=true . And I checked it on another computer. It seems to me Sourceforge servers started sending an invalid file name.
wgetwtf

Comment (gijzelaerr) [reply .github/workflows/windows.yml:18]: ok then it makes sense. note that I added  `--content-disposition `  to make sure wget follows the redirects, but the documentation is a bit minimal on this.

File: .github/workflows/windows.yml (hunk 2 above)
Comment (gijzelaerr) [reply .github/workflows/windows.yml:20]: this is executed in the python-snap7 folder, right? so a `snap7` module should exist.

Comment (nikteliy) [reply .github/workflows/windows.yml:20]: Now I get it(and feel stupid). Thanks for the explanation. I'll move it back.

File: .github/workflows/windows.yml (hunk 1 above)
Comment (nikteliy) [reply .github/workflows/windows.yml:18]: You're right, better keep it. I've checked we can have both options.

File: .github/workflows/windows.yml (hunk 2 above)
Comment (gijzelaerr) [reply .github/workflows/windows.yml]: no worries, you are still being very useful :)

**Pull request merged**